services/web-app/pages/contributing/schema.mdx

```diff
@@ -103,11 +120,14 @@ Libraries are the means to contribute, install, and use one or many assets in pr
 experiences. To index your library with Carbon, create a `carbon.yml` metadata file and place it in
```

Comment:
Suggest starting off defining libraries with the noun (what is is) rather than the verb (what can I do with it).

Libraries are 1:1 with code packages. All coded components, elements, patterns, or functions belong to a library and have a maintainer. Design kits with compatible code also live in libraries.